patches/rrweb@2.0.0-alpha.13.patch

```diff
@@ -1,6 +1,6 @@
 diff --git a/es/rrweb/_virtual/_rollup-plugin-web-worker-loader__helper__browser__createBase64WorkerFactory.js b/es/rrweb/_virtual/_rollup-plugin-web-worker-loader__helper__browser__createBase64WorkerFactory.js
```

Comment:
Worth noting this diff doesn't look 100% correct because GitHub is incorrectly parsing it. I used a text compare tool and only saw differences in spacing & the two changes intentionally removed